python/pyproject.toml

```diff
@@ -22,7 +22,7 @@ runtime_common = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hu
     "outlines>=0.0.44", "modelscope"]
 # xpu is not enabled in public vllm and torch whl,
 # need to follow https://docs.vllm.ai/en/latest/getting_started/xpu-installation.htmlinstall vllm
-srt = ["sglang[runtime_common]", "torch", "vllm==0.6.3.post1"]
+srt = ["sglang[runtime_common]", "torch>=2.5.1", "vllm==0.6.3.post1; python_full_version>'0.0.0'"]
```

Comment:
does vllm 0.6.3.post1 mean the most recent branch? they just updated to 2.5 a few days ago: https://github.com/vllm-project/vllm/pull/9588